core/transfer/src/test/java/org/eclipse/dataspaceconnector/transfer/core/transfer/TransferProcessManagerImplTest.java

```diff
@@ -496,14 +496,15 @@ void verifyCompleted_notAllYetCompleted() throws InterruptedException {
             latch.countDown();
             return List.of(process);
         });
+        when(statusCheckerRegistry.resolve(anyString())).thenReturn((i, l) -> false);
         doThrow(new AssertionError("update() should not be called as process was not updated"))
                 .when(transferProcessStore).update(process);
 
         manager.start();
 
         assertThat(latch.await(TIMEOUT, TimeUnit.SECONDS)).isTrue();
         verify(transferProcessStore, atLeastOnce()).nextForState(anyInt(), anyInt());
-        verify(transferProcessStore, never()).update(any());
+        verify(transferProcessStore).update(argThat(p -> p.getState() == IN_PROGRESS.code()));
```

Comment:
nit: just for readability, if you agree only.
```suggestion
        verify(transferProcessStore).update(argThat(tp -> tp.getState() == IN_PROGRESS.code()));
```

Reply:
Reverted as the other style is used in several other tests